spec/requests/v0/claim_documents_spec.rb

```diff
@@ -16,40 +16,46 @@
     end
 
     it 'uploads a file' do
-      params = { file:, form_id: '21P-527EZ' }
-      expect do
-        post('/v0/claim_documents', params:)
-      end.to change(PersistentAttachment, :count).by(1)
-      expect(response).to have_http_status(:ok)
-      resp = JSON.parse(response.body)
-      expect(resp['data']['attributes'].keys.sort).to eq(%w[confirmation_code name size])
-      expect(PersistentAttachment.last).to be_a(PersistentAttachments::PensionBurial)
+      VCR.use_cassette('uploads/validate_document') do
```

Comment:
why VCR here and not just calling post?

Reply:
@wayne-weibel This VCR is for the PDF validation endpoint, not to mock the POST made to the claim_documents_controller, since we're essentially calling another API endpoint within our endpoint.